File: app/src/main/java/com/arny/myapidemo/models/TestObject.java
package com.arny.myapidemo.models;

import com.arny.arnylib.database.DBProvider;
import org.chalup.microorm.annotations.Column;

import java.io.Serializable;
import java.util.ArrayList;


public class TestObject implements Serializable {
    @Column("id")
    private String id;
    private long dbId;
    @Column("title")
    private String name;

    private ArrayList<GoodItem> goodItems;

	public TestObject(String id, String name) {
        this.id = id;
        this.name = name;
    }

    @Override
    public String toString() {
        return DBProvider.getColumns(this);
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public long getDbId() {
        return dbId;
    }

    public void setDbId(long dbId) {
        this.dbId = dbId;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/com/arny/myapidemo/models/TestObject.kt
package com.arny.myapidemo.models

import android.arch.persistence.room.Entity
import com.arny.arnylib.utils.Utility
import org.chalup.microorm.annotations.Column
import java.io.Serializable

@Entity
open class TestObject(@Column("id") var id: String? = null, @Column("title") var name: String? = null) : Serializable {

    @Column("_id")
    var dbId: Long? = null
    constructor() : this(null, null)
    override fun toString(): String {
        return Utility.getFields(this)
    }
}
